### ISSUE TITLE
Torrent Upload Fails

### ISSUE THREAD

**Issue by BobPNyaa** [NyaaPantsu/nyaa#1750]
I am getting "500 Internal Server Error" anytime I try and upload a new torrent.

**Comment by akuma06** [comment]
Do you still get this? Because I tried to upload and it worked.

**Comment by BobPNyaa** [comment]
Yes, it finally started working about the time you tried. Server issues? It was a pain to keep retyping the upload so a suggestion might still be to allow cut/copy/paste functions in the description box like nyaa.si does.

Sorry for the late reply but the email ended up in my spam box which I did not notice when using the new yahoo mail interface.
BobP
